src/modules/GroupChannel/components/MessageList/index.tsx

```diff
@@ -1,5 +1,5 @@
 import './index.scss';
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useLayoutEffect, useRef, useState } from 'react';
```

Comment:
The work in this file began with reverting https://github.com/gathertown/sendbird-uikit-react/pull/22. I needed the `ref` bound list height to represent purely message height, never empty/loading state heights.

I chose the alternate fix, which is just to call `resolve` even when the `scrollRef` is `null` in `src/modules/GroupChannel/context/hooks/useMessageListScroll.ts`. I think this is okay for reasons mentioned in the added code comment.